builder/dockerfile/copy_windows.go

```diff
@@ -1,23 +1,104 @@
 package dockerfile
 
 import (
+	"bytes"
 	"errors"
+	"fmt"
+	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/Microsoft/go-winio"
 	"github.com/docker/docker/pkg/idtools"
+	"github.com/docker/docker/pkg/reexec"
+	"github.com/docker/docker/pkg/system"
+	"golang.org/x/sys/windows"
 )
 
 var pathBlacklist = map[string]bool{
 	"c:\\":        true,
 	"c:\\windows": true,
 }
 
-func fixPermissions(source, destination string, rootIDs idtools.IDPair, overrideSkip bool) error {
-	// chown is not supported on Windows
+func init() {
+	reexec.Register("windows-fix-permissions", fixPermissionsReexec)
+}
+
+func fixPermissions(source, destination string, identity idtools.Identity, overrideSkip bool) error {
```

Comment:
`_ bool` instead of `overrideSkip bool` so  that it's clear the argument isn't used.

Reply:
I made this change.